packages/webapp/src/hooks/useIsAboveBreakpoint.js

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from 'react';
+
+const useIsAboveBreakpoint = (mqString) => {
+  // this will get a boolean value later, but initialize with null so that
+  // a wrong button will not be shown initially
```

Comment:
It would be great if you could generalize this comment!
Also, license is missing :)

Reply:
Oops! Thank you, just fixed it 🙂